swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/images/AmazonImageHandler.kt

```diff
@@ -194,6 +196,7 @@ class AmazonImageHandler(
         setUsedByLaunchConfigurations(images, params)
         setUsedByInstances(images, params)
         setSeenWithinUnusedThreshold(images)
+        setUsedByLaunchTemplates(images, params)
```

Comment:
Checks if images are referenced by launch templates. If so, the ami would not qualify for deletion.